quadrat/functions.php

```diff
@@ -9,6 +9,17 @@ function quadrat_support() {
 				'/assets/theme.css',
 			)
 		);
+
+		add_theme_support( 'block-nav-menus' );
+
+		// This theme uses wp_nav_menu() in two locations.
+		register_nav_menus(
+			array(
+				'primary' => __( 'Primary Navigation', 'quadrat' ),
+				'footer'  => __( 'Footer Navigation', 'quadrat' ),
```

Comment:
Could we add just the primary location for now or do you see a benefit to having all three locations?

Reply:
I think we need the social one too, to do the Quadrat nav. Not sure about the footer one...

Reply:
The [comps](https://www.figma.com/file/EDoXTvsZt4lMHItzOczFQw/?node-id=443%3A4) suggest the footer is just a WordPress credit, so let's remove that and 🚢 ?

<img width="539" alt="Screen Shot 2021-04-23 at 10 12 55 AM" src="https://user-images.githubusercontent.com/5375500/115884152-733a7480-a403-11eb-81e3-b447a437d9f6.png">

Reply:
Done